### DISCUSSION TITLE
eslint config in a project that embeds pocketbase extensions

### DISCUSSION THREAD

**Discussion by Shmookoff** [pocketbase/pocketbase#3654] [Q&A]
I have a project with the following structure:
```
my-app/
├─ pocketbase/
│  ├─ hooks/
│  ├─ migrations/
│  ├─ pb_data/
│  │  ├─ types.d.ts
├─ src/
│  ├─ app_stuff/
├─ .eslintrc.json
├─ package.json
```
`.eslintrc.json` is set up with rules that are applicable to application files (`src/`).  
When I run `eslint .`, it applies all of those rules to `pocketbase/` the same as `src/`, and in turn generates a long list of errors and warnings that are not applicable to `pocketbase/` part of the project.

I want my `pocketbase/` configuration to be linted with applicable rules.
The question is as follows:
1. Is there a suggested eslint configuration for `pocketbase/`?
2. How do I separate those configurations so that the code is linted with corresponding configs when I run `eslint .` ?

**Comment by ganigeorgiev** [comment]
1) There is no officially recommended eslint/prettier/etc. configuration and users are free to use whatever they prefer.

2) You can define `.eslintignore` file or define the `ignores` or `ignorePatterns` config option (_see also [ESLint Ignore Files](https://eslint.org/docs/latest/use/configure/ignore)_). Alternatively you can have multiple ESLint config files and run the `eslint` command with the [`--config` flag](https://eslint.org/docs/latest/use/command-line-interface#-c---config) (_you can also define the commands as part of your `package.json` "scripts" object_).
